src/client/pps.go

```diff
@@ -174,12 +174,13 @@ func NewPipelineInput(repoName string, glob string) *pps.PipelineInput {
 // CreateJob creates and runs a job in PPS.
 // This function is mostly useful internally, users should generally run work
 // by creating pipelines as well.
-func (c APIClient) CreateJob(pipeline string, outputCommit *pfs.Commit) (*pps.Job, error) {
+func (c APIClient) CreateJob(pipeline string, outputCommit, statsCommit *pfs.Commit) (*pps.Job, error) {
```

Comment:
Will this break current users of our go client library? I don't know how many actual users there are, but I think at least in theory people are supposed to be using this.

Reply:
I'm pretty confident there's no one using this function, since there's basically no correct way to use it, short if writing your own implementation of the worker. But even if they are they just need to pass `nil` for the final argument which isn't that terrible.